gcp/website/blog/content/posts/2024-in-review/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "The Year in Review"
+date: 2024-12-13T04:00:00Z
+draft: false
+author: The OSV Team
+---
+2024 has been an *even more* eventful year for OSV.
+
+<!--more-->
+
+## New ecosystems support
+
+[OSV Schema](https://github.com/ossf/osv-schema) adoption momentum continued, with 2024 being the year of the Linux distributions with four adopting the schema, and now included in our [OSV.dev](https://osv.dev/list) database:
+
+* [Ubuntu](https://openssf.org/blog/2024/06/11/ubuntu-security-notices-now-available-in-osv/)
+* [Chainguard](https://openssf.org/blog/2024/07/03/chainguard-enhances-security-with-osv-advisory-feed/)
+* [Red Hat](https://openssf.org/blog/2024/11/01/red-hats-collaboration-with-the-openssf-and-osv-dev-yields-results-red-hat-security-data-now-available-in-the-osv-format/)
+* SUSE/openSUSE
+
+We also expanded our existing coverage of Debian GNU/Linux, by [including CVE data from their Security Tracker](https://osv.dev/blog/posts/supporting-debian-security-tracker-data/) in our existing CVE record conversion.
+
+Additionally, the [curl project](https://curl.se/) [started contributing vulnerability records](https://osv.dev/blog/posts/announcing-curl-via-rest/).
+
+This has brought the total number of supported ecosystems to 30. The significantly increased coverage of Linux distributions has been very encouraging, and will enable a comprehensive container image scanning story in 2025.
+
+### Impact of the NVD's analysis challenges on Git commit range coverage
+
+Last year, we [announced](https://osv.dev/blog/posts/introducing-broad-c-c++-support/) the expansion of coverage of C/C++ software with Git range coverage of CVEs
```

Comment:
are there any metrics we can call out for 2025 re how much we managed to convert? 

might also be worth highlighting the ingestion improvements we've been making throughout the year? e.g. handling invalid commits + others.

Reply:
> are there any metrics we can call out for 2025 re how much we managed to convert?

Added something in e1ef1c60

Reply:
> might also be worth highlighting the ingestion improvements we've been making throughout the year? e.g. handling invalid commits + others.

Added something in 60510667